core/src/test/scala/integration/kafka/api/LegacyAdminClientTest.scala

```diff
@@ -87,11 +97,11 @@ class LegacyAdminClientTest extends IntegrationTestHarness with Logging {
     consumer.seekToBeginning(Collections.singletonList(tp))
     assertEquals(0L, consumer.position(tp))
 
-    client.deleteRecordsBefore(Map((tp, 5L))).get()
```

Comment:
Do we have a test like this in `AdminClientIntegrationTest`? If not, we should move it there. If so, we should just remove this test.

Reply:
I removed the duplicate delete records tests, and moved the non-duplicate ones over.